news/_posts/translation/2021-06-09-prestaashop-178-translations.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "1.7.8 is ready for translation"
+subtitle: "New features equal new strings"
+date: 2021-06-09 12:00:00
+authors: [JulieVarisellaz]
+icon: icon-world
+tags: [translation, i18n, l10n, 1.7]
+---
+
+## 1.7.8 strings are available for translation
+
+As you probably noticed with the announcement of the beta release a few days ago, PrestaShop 1.7.8 is on its way! And new features also mean new strings. A file containing the new 1.7.8 strings has been added to [Crowdin](https://crowdin.com/project/prestashop-official), and is ready for translation! 
+
+Translators, proofreaders… your project needs you! Feel free to take a look at those new translations and help your language reach 100%. :muscle:
+
+In this article, I will address the key elements of the translation process of this new catalog. 
+ 
+![Help us translate PrestaShop 1.7.8](/assets/images/2021/06/build-help-translate-178.png)
+
+### New strings & completion rates 
+
+If you visited Crowdin in the last few days, you probably noticed that the completion rates for all projects have slightly decreased and that the projects that used to be 100% translated and validated are no longer complete. Don’t worry, that’s completely normal as new untranslated and unapproved strings were added to the 1.7.8 file. In order to reach a 100% completion rate, all the new strings should be translated and validated. 
+
+### Mistakes in the source strings
+
+Even though we always review the wording, some mistakes could have been missed. So, don’t hesitate to let us know if you notice something wrong so that we can correct it. Note that, despite the string freeze, we will wait about 2 or 3 more weeks before adding the corrections to Crowdin. After that, we won’t be able to add any new wording. If you spot a mistake, please tag me (@Julievrz) in the “Comments” section of the string, or [contact me](https://crowdin.com/profile/julievrz) directly!
```

Comment:
_Even though the wording is always reviewed when a pull request is submitted_, what do you think?

Reply:
I like your suggestion, thanks!